### PR TITLE
TSpline Nodes Documentation - Second set

### PR DESCRIPTION
**Pull request by dariaivanciucova**
### Purpose

A second batch of documentation help files for T-Spline nodes.
The In-Depth description follows [ this guidance. ](https://app.mural.co/t/autodesk2145/m/autodesk2145/1692294995308/331cbe5dd536cfa7530d18679cf94f467f9309eb?sender=63628aa0-4092-42ec-b036-49f61f25d8fa)
This batch includes In-Depth, Example image, and .dyn files for the following 38 nodes:

**TSplineInitialSymmetry**

- [x] TSplineInitialSymmetry.ByAxial
- [x] TSplineInitialSymmetry.ByRadial
- [x] TSplineInitialSymmetry.IsRadial
- [x] TSplineInitialSymmetry.RadialSymmetryFaces
- [x] TSplineInitialSymmetry.XAxis
- [x] TSplineInitialSymmetry.YAxis
- [x] TSplineInitialSymmetry.ZAxis

**TSplineReflection**

- [x] TSplineReflection.ByAxial
- [x] TSplineReflection.ByRadial
- [x] TSplineReflection.Axis
- [x] TSplineReflection.IsRadial
- [x] TSplineReflection.Plane
- [x] TSplineReflection.SegmentAngle
- [x] TSplineReflection.SegmentsCount

**TSplineSurface related to Reflection**

- [x] TSplineSurface.Reflections
- [x] TSplineSurface.AddReflections
- [x] TSplineSurface.RemoveReflections

**TSplineSurface Create**

- [x] TSplineSurface.BuildFromLines
- [x] TSplineSurface.ByCombinedTSplineSurfaces
- [x] TSplineSurface.ByNurbsSurfaceCurvature **In-Depth is copied from short description**
- [x]  TSplineSurface.ByNurbsSurfaceUniform **In-Depth is copied from short description**
- [x] TSplineSurface.ByRevolve
- [x] TSplineSurface.BySweep

**TSplineSurface Action**

- [x] TSplineSurface.BevelEdges
- [x] TSplineSurface.CreaseEdges
- [x] TSplineSurface.CreaseVertices
- [x] TSplineSurface.UncreaseVertices
- [x] TSplineSurface.UncreaseEdges
- [x] TSplineSurface.UnweldEdges
- [x] TSplineSurface.UnweldVertices
- [x] TSplineSurface.BridgeEdgesToEdges
- [x] TSplineSurface.BridgeEdgesToFaces
- [x] TSplineSurface.BridgeFacesToEdges
- [x] TSplineSurface.BridgeFacesToFaces
- [x] TSplineSurface.CompressIndexes
- [x] TSplineSurface.DeleteFaces
- [x] TSplineSurface.DeleteEdges
- [x] TSplineSurface.DeleteVertices

Example of In-Depth description:

image

### Declarations

Check these if you believe they are true

- [ ] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated
- [ ] This PR contains no files larger than 50 MB

### Release Notes

For these nodes, the short description is too long and can be moved into the In-Depth description section. 

### Reviewers

(FILL ME IN) Reviewer 1 (If possible, assign the Reviewer for the PR)

(FILL ME IN, optional) Any additional notes to reviewers or testers.

### FYIs

(FILL ME IN, Optional) Names of anyone else you wish to be notified of

### CODE REVIEW

File: doc/distrib/NodeHelpFiles/Autodesk.DesignScript.Geometry.TSpline.TSplineSurface.UnweldEdges.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
-## UnweldEdges - Documentation
-This documentation file is auto generated by NodeDocumentationMarkdownGenerator, Version=2.14.0.3986, Culture=neutral, PublicKeyToken=null.
+## In-Depth
 
-For more information about adding documentation to nodes see https://github.com/DynamoDS/Dynamo/wiki/Create-and-Add-Custom-Documentation-to-Nodes
+In the example below, Unweld operation is performed on row of edges of a TSpline surface. As a result, the vertices of selected edges are disjointed. Unlike Uncrease that creates a sharp transition around the edge while maintaining the connection, Unweld creates a discontinuity. This can be proven by comparing the number of vertices before and after the operation was performed. Any subsequent operations on unwelded edges or vertices will also demonstrate that the surface is disconnected along the unwelded edge.
```
Comment (hwahlstrom) [inline doc/distrib/NodeHelpFiles/Autodesk.DesignScript.Geometry.TSpline.TSplineSurface.UnweldEdges.md:3]: "performed on **a** row of edges..."

Comment (hwahlstrom) [inline doc/distrib/NodeHelpFiles/Autodesk.DesignScript.Geometry.TSpline.TSplineSurface.UnweldEdges.md:3]: "Unlike Uncrease, **which** creates..."

File: doc/distrib/NodeHelpFiles/Autodesk.DesignScript.Geometry.TSpline.TSplineSurface.UnweldVertices.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
-## UnweldVertices - Documentation
-This documentation file is auto generated by NodeDocumentationMarkdownGenerator, Version=2.14.0.3986, Culture=neutral, PublicKeyToken=null.
+## In-Depth
+Similar to `TSplineSurface.UnweldEdges`, this node performs the unweld operation on a set of vertices. As a result, all edges joining at the selected vertex are unwelded. Unlike the Uncrease operation that creates a sharp transition around the vertex while maintaining the connection, Unweld creates a discontinuity.
```
Comment (hwahlstrom) [inline doc/distrib/NodeHelpFiles/Autodesk.DesignScript.Geometry.TSpline.TSplineSurface.UnweldVertices.md:2]: "Unlike the Uncrease operation, **which** creates...."

**Review by hwahlstrom — COMMENTED** [review]
LGTM with small fixes

**Review by QilongTang — APPROVED** [review]

**Pull request merged**